requirements.txt

```diff
@@ -1,4 +1,4 @@
-requests==2.22.0
+requests>=2.20.1
```

Comment:
I am ok with this change, but @supernothing has raised security concerns in the past. That is why we always pin the package version. I would suggest we use a compatible version notation if we choose to introduce this change:
`requests~=2.20`
https://www.python.org/dev/peps/pep-0440/#compatible-release